content/blog/posts/for-the-love-of-the-game.md

```diff
@@ -0,0 +1,109 @@
+
+It's really nice to know that if you need a feature, there may be somebody who's already made it, even if it's not part of the engine already. And if no one's made it and it's not part of the engine, you can make it and it will work as well as anything that was officially done. You can get native support for anything and that's really fantastic.
+
+**To enable some capabilities in the Engine requires the user to change a few lines of code in the code file. How do you feel about the process of installing and/or adding gems, and how could it be improved?**
+
+**Jacob:**  While neither of us have direct, hands-on experience with adding gems and capabilities specifically in O3DE that aren’t built into the Engine, we've done similar things in other engines and programs, so going into a file and changing a few lines of code isn’t a problem. We’ve also read the documentation for how to enable capabilities in O3DE that aren’t already built in, such as [PopcornFX](https://www.popcornfx.com/plugin-o3de/) and [networking](https://docs.o3de.org/docs/user-guide/networking/), and it seems pretty straightforward and simple to do. It’s also pretty typical for programs and software that are relatively new and still being developed, so for us, that's not a problem. Integrating these capabilities into the Engine would obviously save time, but it's not a huge deal.
+
+**We’ve heard about some great upsides that O3DE offers. Is there anything you found that could be enhanced?**
+
+**Phoenix:**  I was working on some player stuff based on the [PhysX](https://www.o3de.org/docs/user-guide/interactivity/physics/) player controller in O3DE, and
+
+I ran into some issues. The PhysX player controller is more cinematic than I would want—it’s not really directly physics-based, despite the name—and I wanted to do something more in that space, but couldn't really find the resources to do that.
+
+**Jacob:**  The capability of automatically building a project when you’re finished creating it would be a nice feature. Other than that, there aren’t many more improvements I can think of because it's all pretty intuitive and easy on the eyes.
+
+**Are you currently contributing to O3DE, or would you like to?**
+
+**Jacob:**  Yeah, we've done a little bit, but nothing crazy. We’ve helped where we can, but balancing work and life makes this a bit difficult. I got involved briefly in the Outreach committee. I also did a little contributing to the SIG Docs, just pointing out things on the website or typos or you know little things here and there.
+
+**For newcomers who are getting started with O3DE, are there any suggestions or recommendations you might have for them?**
+
+**Jacob:**  My first recommendation would be to pull up the [documentation](https://www.o3de.org/docs/) and get started because, like we said, the onboarding is smooth and easy to understand. If you have questions, it's easy to go into [Discord](http://discord.com/invite/o3de) and ask.
```

Comment:
Docs are internal, Change 
`[documentation](https://www.o3de.org/docs/) `
to
`[documentation](/docs/) `